Makefile

```diff
@@ -22,6 +22,8 @@ dev: start-support
 	@yarn dev
 dev-workspace-host: start-support
 	@yarn dev-workspace-host
+dev-all: start-support
+	@$(MAKE) -s -j2 dev dev-workspace-host
```

Comment:
`-s` is not needed here because `dev` and `dev-workspace-host` commands start with a `@`.

Reply:
Yeah, I think I added `-s` before I put `@` in the front. Confirmed, it's not needed.

Reply:
Ah, it is still useful:

```
bash-5.2# make dev-all
make[1]: Entering directory '/PrairieLearn'
[nodemon] 3.1.0
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: js,ts,json,sql
[nodemon] starting `yarn dev:no-watch`
[nodemon] 3.1.0
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: js,mjs,cjs,json
[nodemon] starting `yarn dev:no-watch`
info: Workspace server listening on port 8081
info: Workspace host ready
info: PrairieLearn server ready, press Control-C to quit
info: Go to https://localhost:443
```